doc/UserGuide.md

```diff
@@ -45,7 +45,7 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > Persons can have any number of tags (including 0)
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
+* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01, 123456`
```

Comment:
user guide should also indicate the format of the address now supported explicitly; not just an example